mmv1/third_party/terraform/provider/provider_mmv1_resources.go.tmpl

```diff
@@ -169,6 +169,7 @@ var handwrittenDatasources = map[string]*schema.Resource{
 	"google_monitoring_uptime_check_ips":               monitoring.DataSourceGoogleMonitoringUptimeCheckIps(),
 	"google_netblock_ip_ranges":                        resourcemanager.DataSourceGoogleNetblockIpRanges(),
 	"google_oracle_database_db_servers":                oracledatabase.DataSourceOracleDatabaseDbServers(),
+	"google_oracle_database_cloud_exadata_infrastructures": oracledatabase.DataSourceOracleDatabaseCloudExadataInfrastructures(),
```

Comment:
What is the difference of `google_oracle_database_cloud_exadata_infrastructures` and `google_oracle_database_cloud_exadata_infrastructure`?

Reply:
It's for get and list API separately.